File: VB/DataLayer/Data/XtraGridTutorialDBContext.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports System.Data.Entity
Imports DataLayer.Data.Classes

Namespace DataLayer.Data
    Public Class XtraGridTutorialDBContext
        Inherits DbContext

        Public Sub New()
            MyBase.New("name=XtraGridTutorialDBConnectionString")
                Me.Configuration.LazyLoadingEnabled = True
        End Sub

        Public Property Accounts() As DbSet(Of Account)
        Public Property Categories() As DbSet(Of Category)
        Public Property Transactions() As DbSet(Of Transaction)
    End Class
End Namespace

File: CS/DataLayer/Data/XtraGridTutorialDBContext.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Data.Entity;
using DataLayer.Data.Classes;

namespace DataLayer.Data
{
    public class XtraGridTutorialDBContext : DbContext {
        public XtraGridTutorialDBContext()
            : base("name=XtraGridTutorialDBConnectionString") {
                this.Configuration.LazyLoadingEnabled = true; 
        }

        public DbSet<Account> Accounts { get; set; }
        public DbSet<Category> Categories { get; set; }
        public DbSet<Transaction> Transactions { get; set; }
    }
}
